__init__.py

```diff
@@ -15,7 +14,8 @@ def Run(self):
         kicad_pcb = pcbnew.GetBoard()
         with open(kicad_pcb.GetFileName().replace('.kicad_pcb', '.brd'), 'wt') as brd_file:
             convert(kicad_pcb, brd_file)
-
+        obdata_filename = kicad_pcb.GetFileName().replace('.kicad_pcb', '.obdata')
+        generate_obdata(kicad_pcb, obdata_filename)
```

Comment:
The `generate_obdata` function should take a file object, not a filename. (Like `convert`).

Rename `convert` to `convert_brd` and `generate_obdata` to `convert_obdata`.

Reply:
Ok I've updated the script, let me know if this version is okay in regards to this.